src/libraries/Common/src/System/Security/Cryptography/ECDsaAndroid.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Security.Cryptography
 {
+#if INTERNAL_ASYMMETRIC_IMPLEMENTATIONS
```

Comment:
I followed the existing pattern I saw for OpenSSL and OSX implementations using `INTERNAL_ASYMMETRIC_IMPLEMENTATIONS`. I did notice that for ECDiffieHellman, we have a separate ECDiffieHellman.Create.* file that has these static functions, so we don't have them ifdef-ed out in Common. Not sure if there's a reason we can't or don't want the same for the others?

Reply:
For the Windows types the #define also controls whether they're nested types or top-level types, which is where the pattern started.

I'm cool with an ECDsa.Create.Android.cs instead of \#if.